### ISSUE TITLE
Fedora MATE-Compiz spin need to include missing packages from full DVD

### ISSUE THREAD

**Issue by TruboXL** [mate-desktop/mate-desktop#215]
Don't really know where to report this one @.@
This is more related to the Fedora MATE side than the generic MATE as a whole.
IRC doesn't seem to be the most effective way of communicating...
And I don't know you guys as upstream can do anything about what ends up at the Fedora spin (or is it?)

For Fedora MATE SIG team, please check this bug's personal note to you guys...
https://bugzilla.redhat.com/show_bug.cgi?id=1310490

TLDR: ntp is missing in the live DVD and possibly there are a few more missing packages out there that may render the live DVD not fully functional.

**Comment by raveit65** [comment]
Not a upstream issue, why not installing the missing package for yourself?

**Comment by TruboXL** [comment]
But then again (who?) just shipping an unfinished spin to public and that is not really acceptable.

Of course I can add the missing packages however this feature is present in the spin but functionally **partial** and not like that it isn't part of the spin. **It is a crucial part of the spin.**

Also, when I do install from the live DVD, anaconda given me the option to have network sync time feature on by default if I connect to the Internet

The installer deliberately says network sync time is available however the end product (the install) is not.

So yeah am I talking to right people here? The ones who made the spin? :(

**Comment by raveit65** [comment]
Please file out a report at redhat bugzilla package mate-desktop.
There we can discuss about missing packages.
If you have problems with anaconda file out a report against anaconda.
The fedora installer is not maintained by me or MATE.

**Comment by TruboXL** [comment]
Ok noted.

Seem to have some progress over system-config-date showing blank network time servers even though anaconda added them to /etc/chrony.conf

Case closed for here as ntpd is already replaced by chronyd long time ago.
So missing ntpd will not affect time syncing.
